src/pipx/commands/run.py

```diff
@@ -140,7 +144,7 @@ def _download_and_run(
         is_main_package=True,
     )
 
-    if not (venv.bin_path / app).exists():
+    if not (venv.bin_path / app_filename).exists():
```

Comment:
The following `PipxError` seems to be the only remaining necessary use of `app`.  Is it useful to have both `app` and `app_filename` just so that the error message doesn't suffix the app with ".exe"?  Or am I missing another use for the two separate variables?

Reply:
OK, I see your other comment.  So it seems this is mainly a setup for your other PR?

Reply:
I’m also thinking maybe eventually the `.exe` existence checks should be replaced by `shutil.which()` (and `app_filename` can be dropped entirely after that), but that needs a wider discussion and I’m probably not going to spend much energy on it unless someone complains about pipx assuming `.exe`.

Reply:
OK.  This change seems innocuous to me, so if it helps you to merge this it's fine with me.